blender-mcp.md

```
@@ -0,0 +1,154 @@
+- setup a venv
+- activate the venv
+- `pip install .`
+- `Get-Command blender-mcp` to find the path to the executable, which should env with `...venv\Scripts\blender-mcp.exe` on Windows, e.g.
+
+C:\Users\username\src\blender\blender_mcp\venv\Scripts\blender-mcp.exe
+
+Configure Ai Playground (as a developer):
+
+in mcpServers.ts, extend the existing `mcpServers` object with a new entry, e.g.
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)